### ISSUE TITLE
Bot disconnects from channel when someone else disconnects (review branch)

### ISSUE THREAD

**Issue by gabrielboliveira** [Just-Some-Bots/MusicBot#2032]
Please tick all applicable boxes.

- [x] I am using Python 3.5.3 or higher (run `python --version` on the command line)
- [ ] I have followed the [official guides](https://github.com/Just-Some-Bots/MusicBot/wiki) to install the bot for my system
- [ ] I have updated my dependencies to the latest version using the appropriate `update` script

Which version are you using?

- [ ] The latest `master` version (release-260419)
- [x] The latest `review` version

What type of issue are you creating?

- [x] Bug
- [ ] Feature request
- [ ] Question

----

### Description of issue

Using the review Docker image with docker-compose, when anyone disconnects from a channel, the bot disconnect as well. You need to manually summon it back again.

### Steps to reproduce

1. Create docker-compose.yml as below:

```yml
version: '3.7'
services:

  musicbot:
    image: justsomebots/musicbot:review
    restart: unless-stopped
    volumes:
      - ./config:/usr/src/musicbot/config
```
2. Copy config folder to same folder as docker-compose.yml
3. Run with `docker-compose up -d`
4. Join a channel and summon the bot
5. Ask someone else to join same channel
6. When someone disconnects, the bot will leave as well.

Side note: if the person just moves channel, the bot won't leave. Just when disconnecting.

Also, it doesn't matter the channel. It does not need to be the same channel the bot is in.

Eg: bot is in channel 1 with 2 people, person 3 join channel 2 and leaves, the bot will leave from channel 1

----

### Log file

[musicbot.log](https://github.com/Just-Some-Bots/MusicBot/files/4355998/musicbot.log)

**Comment by AleanaA** [comment]
Resolved in #2033

**Comment by gabrielboliveira** [comment]
Thanks! Confirmed as fixed
